SPECS/iptables/iptables.spec

```
@@ -96,41 +100,64 @@ rm -rf %{buildroot}/*
 %{_mandir}/man3/*
 
 %changelog
-* Sat May 09 00:21:34 PST 2020 Nick Samson <nisamson@microsoft.com> - 1.8.3-3
-- Added %%license line automatically
+* Wed Oct 21 2020 Rachel Menge <rachelmenge@microsoft.com> - 1.8.3-5
+- Modified the ip4, ip6 to match rules of other distros
+
+*   Thu Oct 15 2020 Rachel Menge <rachelmenge@microsoft.com> - 1.8.3-4
+-   Use Autosetup and removed old sha1 define
+-   Modified the ip4, ip6 to default to ACCEPT all connections 
+
+*   Sat May 09 00:21:34 PST 2020 Nick Samson <nisamson@microsoft.com> - 1.8.3-3
```

I've seen people fixing these lines with the time/timezone information whenever they touch a spec file.  Apparently those were added by a tool and the formatting was wrong.  Not a big deal, though, but if you end up changing the file, please consider this change, too:

```suggestion
*   Sat May 09 2020 Nick Samson <nisamson@microsoft.com> - 1.8.3-3
```